unicodetools/org/unicode/propstest/CheckEmojiProps2.java

```diff
@@ -0,0 +1,30 @@
+package org.unicode.propstest;
+
+import org.unicode.props.IndexUnicodeProperties;
+import org.unicode.props.UcdProperty;
+import org.unicode.props.UcdPropertyValues.Binary;
+
+import com.google.common.collect.Sets;
+import com.ibm.icu.text.UnicodeSet;
+
+/**
+ * PR #44 added this class, but it was obviously added under the same name with
+ * different code at another time. Re-adding under a different name.
+ */
+public class CheckEmojiProps2 {
```

Comment:
@macchiati this was created with the same name as another class, with different content. So I disambiguated them.

Reply:
lgtm